client_cli/README.md

```diff
@@ -1,35 +1,41 @@
 # Iroha CLI Client
 
-## Description
+With Iroha CLI Client, you can interact with Iroha Peers Web API without directly using the network.
+It's a "light" client, meaning it only converts Command Line Interface commands into Iroha Web API Network Requests.
 
-Iroha CLI Client provides an ability to interact with Iroha Peers Web API without direct network usage.
-It's a "light" client which only converts Command Line Interface commands into Iroha Web API Network Requests.
+## Features
 
-### Features
-
-* Iroha CLI Client can submit Transactions with your Iroha Special Instructions to Iroha Peers
-* Iroha CLI Client can send Requests with your Queries to Iroha Peers
+* Submit Transactions with your Iroha Special Instructions to Iroha Peers
+* Send Requests with your Queries to Iroha Peers
 
 ## Installation
 
-### Requirements
-
-* [Rust](https://www.rust-lang.org/learn/get-started)
+**Requirements:** a working [Rust toolchain](https://www.rust-lang.org/learn/get-started) installed.
 
-### Build
+Build Iroha and its binaries:
 
 ```bash
 cargo build
 ```
 
-### Artifact
+It will produce `iroha_client_cli` executable file for Unix and `iroha_client_cli.exe` executable file for Windows.
+
+## Usage Examples
 
-`iroha_client_cli` executable file for Unix and `iroha_client_cli.exe` executable file for Windows will appear.
-All examples below are Unix oriented, to run them on Windows replace `./iroha_client_cli` with `iroha_client_cli.exe`.
+:grey_exclamation: All examples below are Unix-oriented. To run them on Windows, replace `./iroha_client_cli` with `iroha_client_cli.exe`.
```

Comment:
```suggestion
:grey_exclamation: All examples below are Unix-oriented. If you're on Windows, we would highly encourage you to consider using WSL, as most documentation assumes a POSIX-like shell running on your system. Please be advised that the syntax of commands may change more than just executing `iroha_client_cli.exe`  instead of `iroha_client_cli`. 
```